src/Catalyst.Core.Modules.Dfs/DfsModule.cs

```diff
@@ -21,20 +21,168 @@
+
+            builder.RegisterType<PinApi>().As<IPinApi>().SingleInstance()
+               .OnActivated(e => e.Instance.BlockApi = e.Context.Resolve<IBlockApi>());
+
+            builder.RegisterType<BitSwapApi>().As<IBitSwapApi>().SingleInstance();
+            builder.RegisterType<BlockRepositoryApi>().As<IBlockRepositoryApi>().SingleInstance();
+            builder.RegisterType<BootstrapApi>().As<IBootstrapApi>().SingleInstance();
+            builder.RegisterType<ConfigApi>().As<IConfigApi>().SingleInstance();
+            builder.RegisterType<DagApi>().As<IDagApi>().SingleInstance();
+            builder.RegisterType<DnsApi>().As<IDnsApi>().SingleInstance();
+            builder.RegisterType<UnixFsApi>().As<IUnixFsApi>().SingleInstance();
+            builder.RegisterType<KeyApi>().As<IKeyApi>().SingleInstance();
+            builder.RegisterType<NameApi>().As<INameApi>().SingleInstance();
+            builder.RegisterType<ObjectApi>().As<IObjectApi>().SingleInstance();
+            builder.RegisterType<PubSubApi>().As<IPubSubApi>().SingleInstance();
+            builder.RegisterType<StatsApi>().As<IStatsApi>().SingleInstance();
+            builder.RegisterType<SwarmApi>().As<ISwarmApi>().SingleInstance();
+            builder.RegisterType<DhtApi>().As<IDhtApi>().SingleInstance();
+            builder.RegisterType<MigrationManager>().As<IMigrationManager>().SingleInstance();
             builder.RegisterType<DeltaDfsReader>().As<IDeltaDfsReader>().SingleInstance();
-            builder.RegisterType<IpfsAdapter>().As<IIpfsAdapter>().As<ICoreApi>().SingleInstance();
-            builder.RegisterType<Dfs>().As<IDfs>().SingleInstance();
-        }  
+
+            builder.RegisterType<DfsState>().As<DfsState>().SingleInstance();
+        }
+
+        protected override void LoadService(ContainerBuilder builder)
+        {
+            builder.RegisterType<DfsService>()
+               .As<IDfsService>()
+               .SingleInstance();
+
+            builder.RegisterType<BitSwapService>()
+               .As<BitSwapService>()
+               .SingleInstance();
+
+            builder.RegisterBuildCallback(async x =>
+            {
+                //Log.Debug("Building bitswap service");
+                //Log.Debug("Built bitswap service");
+
+                var localPeer = x.Resolve<Peer>();
+                var swarmService = x.Resolve<SwarmService>();
+
+                var bitSwapService = x.Resolve<BitSwapService>();
+                bitSwapService.SwarmService = swarmService;
+                bitSwapService.BlockService = x.Resolve<IBlockApi>();
+
+                var pubService = x.Resolve<PubSubService>();
+                pubService.LocalPeer = localPeer;
+                pubService.Routers.Add(new FloodRouter
+                {
+                    SwarmService = swarmService
+                });
+
+                //var keyApi = x.Resolve<IKeyApi>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)